src/app/project/view-users-projects/view-users-projects.component.scss

```diff
@@ -115,15 +114,106 @@
   #aa {
     position: relative;
     margin-left: 35%;
-    // display:inline;
     float:left;
   }
   
   #aaa {
-    // display:inline;
     float:left;
     font-family: "Courier New", sans-serif;
     margin-top: 30px;
     font-size: 15px;
     margin-left: 30px;
-  }
\ No newline at end of file
+  }
+
+  .mobile-label {
+    display: none;
+  } 
+  
+  @media screen and (max-width: 960px) {
+    
+    #project-image
+    {
+      min-width: 25vh;
```

Comment:
Fix unconventional formatting issues.

Reply:
Identifiers a, aa, and aaa also used here. Should be evaluated for better naming.